### ISSUE TITLE
Regarding the issue of model performance testing

### ISSUE THREAD

**Issue by Feecuin** [eduardzamfir/MoCE-IR#9]
Dear Author, I re-trained your model according to the instructions in the Readme. I found that when dealing with three types of degraded weights, only one degradation achieved the desired result, while the other two degraded cases had no effect at all. Why is this?

**Comment by eduardzamfir** [comment]
Hi,
Can you please give me more info about what’s going wrong? When using the pretrained models do you achieve the same results?

**Comment by Feecuin** [comment]
> Hi, Can you please give me more info about what’s going wrong? When using the pretrained models do you achieve the same results?

To be specific, I'm not sure exactly where the problem lies. I retrained the weights for the three datasets (rain, fog, and snow) using your network. However, during the testing, only the fog removal was effective; moreover, when using the pre-trained weights for testing, none of the three types of degradation could be removed.

**Comment by Feecuin** [comment]
> > Hi, Can you please give me more info about what’s going wrong? When using the pretrained models do you achieve the same results?
> 
> To be specific, I'm not sure exactly where the problem lies. I retrained the weights for the three datasets (rain, fog, and snow) using your network. However, during the testing, only the fog removal was effective; moreover, when using the pre-trained weights for testing, none of the three types of degradation could be removed.

I trained according to the Readme. Was there any error in the training process?

**Comment by eduardzamfir** [comment]
when using the pretrained weights, did you test on the official benchmarks (SOTS, CSD68 and Rain100L)? Can you be more specific, what does "not effective" mean?

**Comment by Feecuin** [comment]
> when using the pretrained weights, did you test on the official benchmarks (SOTS, CSD68 and Rain100L)? Can you be more specific, what does "not effective" mean?

I used my own constructed dataset, and the training was also based on this dataset.  The term "no effective" means that it is impossible to remove the rain, fog, snow, or other distractions from the original image.

**Comment by eduardzamfir** [comment]
For the pretrained weights, it's not too surprising if the results aren't great on OOD data, since the model isn't primarily designed for blind restoration.

As for your custom dataset, it might help to lower the FFT loss weight, maybe try reducing it to 0.01 or even removing it entirely. Alternatively, you could test with some of the standard datasets first, just to make sure everything is working as expected.

Let me know how it goes and if you see any improvement!
